### PR TITLE
[MIG] testserver

### PR DESCRIPTION
**Pull request by KKamaa**

### CODE REVIEW

**Review by thomaspaulb — APPROVED** [review]

File: testserver/__manifest__.py [hunk 1]
```diff
@@ -4,19 +4,13 @@
...
-    'description': """
-Mark this server as a testserver!
-=================================
-So that it is not confused with the real one
-    """,
     'author': 'Sunflower IT',
     'website': 'http://sunflowerweb.nl',
     'category': 'Specific Industry Applications',
     'sequence': 0,
     'depends': ['base', 'web'],
```
Comment (thomaspaulb) [inline testserver/__manifest__.py]: Dont have to depend on base, [web already does](https://github.com/OCA/OCB/blob/10.0/addons/web/__manifest__.py#L15)

File: testserver/static/src/css/testserver.css [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
+#oe_main_menu_navbar.navbar-inverse {
+    background-color: #040 !important;
 }
 
-.oe_logo {
-  background-image: url('/testserver/static/src/img/testserver.jpg') !important;
-  background-size: cover !important;
+.o_sub_menu_logo {
+    background-image: url('/testserver/static/src/img/testserver.jpg') !important;
+    background-size: cover !important;
```
Comment (thomaspaulb) [inline testserver/static/src/css/testserver.css]: Could you change "cover" to "contain" ? So the text "TESTSERVER" is always visible fully

**Review by thomaspaulb — CHANGES_REQUESTED** [review]

File: testserver/__manifest__.py (hunk 1 above)
Comment (KKamaa) [reply testserver/__manifest__.py]: fixed

File: testserver/static/src/css/testserver.css (hunk 2 above)
Comment (KKamaa) [reply testserver/static/src/css/testserver.css]: fixed

**Pull request closed without merging**